multicore_runtime/wsgi_test.py

```diff
@@ -24,11 +25,20 @@
 
 from mock import MagicMock
 from mock import patch
+from werkzeug import http
 from werkzeug import test
 from werkzeug import wrappers
 
 from google.appengine.api import appinfo
 
+class FakeDatetime(datetime.datetime):
```

Comment:
We should be able to use the mock library to mock datetime.datetime.now(), instead of overriding it manually like this. That will ensure our mock is not too broad and doesn't affect tests we don't want it to, and that multiple mocks can coexist peacefully on the same module down the road.